### PR TITLE
Add django to the project

### PR DESCRIPTION
**Pull request by sadeghesfahani**
the project needed django for backend

### CODE REVIEW

**Comment by sadeghesfahani** [comment]
ok

**Comment by Smrazavi1991** [comment]
hi this is for test

**Comment by sadeghesfahani** [comment]
thanks for test

File: app/app/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
```
Comment (Smrazavi1991) [inline app/app/settings.py:16]: what does this line do?

**Review by Smrazavi1991 — APPROVED** [review]

**Pull request merged**